proxy_map.go

```diff
@@ -98,6 +98,89 @@ Once the lock context is created, it can be used to lock/unlock entries and used
 As mentioned before, lock context is a regular context.Context which carry a special lock ID.
 You can pass any context.Context to any Map function, but in that case lock ownership between operations using the same hazelcast.Client instance is not possible.
 
+Using the Near Cache
+
+Map entries in Hazelcast are partitioned across the cluster members.
+Hazelcast clients do not have local data at all.
+Suppose you read the key k a number of times from a Hazelcast client or k is owned by another member in your cluster.
+Then each map.Get(k) will be a remote operation, which creates a lot of network trips.
+If you have a data structure that is mostly read, then you should consider creating a local Near Cache, so that reads are sped up and less network traffic is created.
```

Comment:
Typo
```suggestion
If you have a data structure that is mostly read, then you should consider creating a local Near Cache, so that reads are speed up and less network traffic is created.
```

Reply:
I took most of the text straight from the Hazelcast docs. I think `sped up` is correct: https://forum.wordreference.com/threads/speed-sped-speeded.858459/

Reply:
Didn't know that, thanks.